### PR TITLE
Dynamic hash

### PR DESCRIPTION
**Pull request by stefvanhouten**
Add support for multiple CookieHashes. 
When creating/updating a cookie check if the hash is deprecated, if so use a random non-deprecated hash to update/create a new cookie.

### CODE REVIEW

File: uweb3/libs/sqltalk/safe_cookie/connection.py [hunk 1]
```diff
@@ -6,6 +6,11 @@ def __init__(self, request, cookies, cookie_salt, autocommit=True, **kwargs):
         self.cookie_salt = cookie_salt
         self.uncommitted_cookies = []
 
+        if kwargs.pop("debug", False):
```
Comment (underdarknl) [inline uweb3/libs/sqltalk/safe_cookie/connection.py:9]: `self.debug = kwargs.pop("debug", False)` should be enough

**Review by underdarknl — APPROVED** [review]
tiny change, but looks excellent

File: uweb3/libs/sqltalk/safe_cookie/connection.py (hunk 1 above)
Comment (stefvanhouten) [reply uweb3/libs/sqltalk/safe_cookie/connection.py]: Now defaults to True.

**Comment by stefvanhouten** [comment]
Refactored the test_model file into smaller chunks and moved them into their own folder.

**Pull request merged**